src/components/Tag/Tag.module.scss

```
@@ -0,0 +1,10 @@
+.tag {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  gap: 0.75rem;
+  background: $color-orange-ffa500;
```

```suggestion
  background: $color-orange-ffa500;
```
이 css가 적용되는 경우도 있나요?? 인라인 css가 더 우선순위 아닌가요??

---

아 삭제해야 하는데 안했네요 ㅎㅎ.. 수정하겠습니다!